### PR TITLE
9053 ga events

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9053)

This is the PR to implement the two GA events for the gating API call. THIS IS ONLY FOR THOSE TWO CALLS, NOT FOR ALL THE GA CALLS ON THE 686 :-)

## Testing done
Tested that events fire using dataslayer on Firefox and Google Tag Manager on Chrome

## Acceptance criteria
- [x] Events are accurately firing for the API call

### CODE REVIEW

File: src/applications/disability-benefits/686c-674/config/form.js [hunk 1]
```diff
@@ -380,14 +378,16 @@ const formConfig = {
...
+            isChapterFieldRequired(formData, TASK_KEYS.report674),
           title: 'Information needed to add a student 18 to 23 years old',
           path: 'report-674-student-income-information',
           uiSchema: studentIncomeInformation.uiSchema,
           schema: studentIncomeInformation.schema,
         },
         studentNetworthInformation: {
-          depends: () => false,
+          depends: formData =>
+            isChapterFieldRequired(formData, TASK_KEYS.report674),
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/form.js:390]: These need to be reverted.

File: src/applications/disability-benefits/686c-674/config/form.js [hunk 2]
```diff
@@ -68,15 +68,13 @@ const formConfig = {
...
-  getHelp: GetFormHelp,
+  verifyRequiredPrefill: true,
   savedFormMessages: {
     notFound: 'Please start over to apply for declare or remove a dependent.',
     noAuth:
       'Please sign in again to continue your application for declare or remove a dependent.',
   },
-  title: 'Add or remove dependents from your VA benefits',
-  subTitle: 'VA Form 21-686c (and 21-674)',
+  title: 'New 686',
```
Comment (micahchiang) [inline src/applications/disability-benefits/686c-674/config/form.js:77]: I think these need to be reverted.

**Review by micahchiang — APPROVED** [review]
I must have missed this pull request while I was out on PTO. Really sorry about that! Approved, but the changes to the 686 form config need to be discarded before this gets merged in I believe.

File: src/applications/disability-benefits/686c-674/config/form.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/form.js]: fixed

File: src/applications/disability-benefits/686c-674/config/form.js (hunk 2 above)
Comment (cohnjesse) [reply src/applications/disability-benefits/686c-674/config/form.js]: fixed

**Review by micahchiang — APPROVED** [review]

**Pull request merged**